Documentation/writing-tizen-csharp-apps.md

```diff
@@ -0,0 +1,131 @@
+# Writing a Tizen .NET Application
+
+This document provides you how to write a Tizen .NET Application with Machine Learning APIs.
+Since Tizen 5.5 (not released yet), Machine Learning Inference functionality has been provided on Mobile, Wearable and TV profile.
+
+## Installing Visual Studio Tools for Tizen
+In order to use this functionality, you need to install Visual Studio Tools for Tizen and Tizen SDK version 5.5 or higher. You can find the detailed guide for this at the below link.
```

Comment:
There is no Tizen SDK. It's Tizen Studio.
Besides, there won't be Tizen Studio 5.5 in 2019 or 2020. The most recent version of Tizen studio as of 2019-10-18 is 3.3 (Studio version goes independently from Tizen version.)

Reply:
According to the [official guide](https://developer.tizen.org/development/visual-studio-tools-tizen/installing-visual-studio-tools-tizen#install) for `Installing Visual Studio Tools for Tizen`, developers should install both 'Tools for Tizen' and 'Tizen Baseline SDK'. I found out that there is no specific version for SDK. So I will fix it as below.

> In order to use this functionality, you need to install Visual Studio Tools for Tizen and Tizen SDK. You can find the detailed guide for this at the below link.